src/components/PersonLink.tsx

```diff
@@ -0,0 +1,65 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const { slugParam } = useParams();
+
+  const {
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    name,
+    slug,
+  } = person;
+  const hasMother = people.find(p => p.name === person.motherName);
+  const hasFather = people.find(p => p.name === person.fatherName);
```

Comment:
these variables are not boolean so let's use more self-descriptive names
```suggestion
  const mother = people.find(p => p.name === person.motherName);
  const father = people.find(p => p.name === person.fatherName);
```